texts/depositing/compliance-notice.md

```diff
@@ -1,9 +1,5 @@
 ---
-title: Compliance
+title: Data Overview
+redirect: Review data
 ---
 
-You have **{{amount}}%** outputs are non-compliant according to
```

Comment:
As I wrote above, please create a separate text scope for this change. We louse this information at the moment but we should not. It should stay on the page.

I would display this below the page title in grey text. Let me know if you need designer help with this.

Reply:
Please let me know where shoudl this card be placed or add it on your own. Whatever makes it faster. :)

Reply:
Sorry I missed the second paragraph. I will add it below the title.

Reply:
This is not what I meant. Could you please rever all changes in this card and keep it unused at the moment?